runtime/errors.go

```diff
@@ -216,3 +217,29 @@ func (e *TransactionParameterTypeNotStorableError) Error() string {
 		e.Type.QualifiedString(),
 	)
 }
+
+// ExtendedParsingCheckingError is a special error which aids in debugging checking problems
+// by providing extra information about the state of the environment
+// Separate package to prevent cyclic imports with checker tests
```

Comment:
Not quite sure I understand this line, can you elaborate?

Reply:
Does this clarify cyclic imports enough? https://github.com/onflow/cadence/pull/451/files#diff-74ffa81b6938856f695ba112a721503dfc5ff2e852bc65d79d15ee4868f4bb97R4784-R4791